src/drivers/sf0x/sf0x.cpp

```diff
@@ -262,7 +262,7 @@ SF0X::init()
 {
 	/* do regular cdev init */
 	if (CDev::init() != OK) {
-		goto out;
```

Comment:
Are we freeing all allocated fds correctly? (before the change and in particular after?)